language/stdlib/modules/LibraSystem.move

```diff
@@ -451,13 +466,19 @@ module LibraSystem {
         if (!ValidatorConfig::is_valid(validator_info.addr)) {
             return false
         };
+        // This should is ensured by the code calling this function, but left here for safety.
```

Comment:
I worry that redundant checks like this can actually reduce safety by causing confusion over whose responsibility it is to enforce the invariant... For example, someone might delete the other check, continue to see tests passing, and think that it is ok (when in fact, the other check is the "real" one).

Thus, my preference would be to omit the check here and keep only the one at 276.

Reply:
A rule of thumb I use for check redundancy: if you comment out any check, a test should fail. Otherwise, the check is redundant (or you don't have enough tests :) ).

Reply:
Totally makes sense!